Sources/AppearanceManager.swift

```diff
@@ -81,4 +81,23 @@ extension UINavigationController {
     override open var childForStatusBarStyle: UIViewController? {
         return self.topViewController
     }
+
+    override open func viewDidLoad() {
```

Comment:
The most important part: fix navigationController behavior - react on theme change